src/javaRosa.js

```diff
@@ -792,7 +792,12 @@ define([
     };
 
     var itextMediaBlock = function (mug, options) {
-        var block = itextConfigurableBlock(mug, options);
+        var block = itextConfigurableBlock(mug, options),
+            pathPrefix = options.pathPrefix;
+
+        if(!_.isString(options.pathPrefix)) {
```

Comment:
Eh. I suppose that technically concatenating strings with booleans and numbers is legal in javascript, but since this is getting appended in a uri a string would be best and more clearly states intent